lib/collective/collectors/trackjs.rb

```diff
@@ -0,0 +1,110 @@
+require 'pp'
+
+module Collective::Collectors
+  class TrackJS < Collective::Collector
+    requires :faraday
+    requires :faraday_middleware
+    requires :json
+
+    resolution '60s'
+
+    def initialize(lastSeenId = nil, options)
+      @lastSeenId = lastSeenId
+      super(options)
+    end
+
+    collect do
+      instrument_errors
+    end
+
+    private
+
+    def instrument_errors
+      count = 0
+      application = ""
+      paged("/#{customer_id}/v1/errors").each do |error|
+        application = error['application'] if count == 0
+        count += 1
```

Comment:
Also, do you know if the "Time seen" header is a property of the error or are they summing up the errors by the Message? Seems like the latter, but would be good to confirm.